scrapy/core/http2/agent.py

```diff
@@ -95,16 +95,18 @@ def close_connections(self) -> None:
 
 class H2Agent:
     def __init__(
-        self, reactor: ReactorBase, pool: H2ConnectionPool,
-        context_factory=BrowserLikePolicyForHTTPS(),
-        connect_timeout: Optional[float] = None, bind_address: Optional[bytes] = None
+        self,
+        reactor: ReactorBase,
+        pool: H2ConnectionPool,
+        context_factory: Type[BrowserLikePolicyForHTTPS] = BrowserLikePolicyForHTTPS(),
```

Comment:
I’m confused. The parameter is an instance, the typing data says it’s the class itself.

Reply:
Oh, my bad